readthedocs/projects/managers.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+from django.db import models
+from .ssh import generate_public_from_private_key
+
+
+class SSHKeyManager(models.Manager):
```

Comment:
We've moved from Manager class to QuerySet class, as you can't chain queries when just using a manager. The project models have some examples. QuerySet is not immediately useful for this case, but for consistency it's better.